drag _Archiver-A_ here

```diff
@@ -1 +1 @@
-/boot/home/config/add-ons/Tracker
\ No newline at end of file
+/boot/system/settings/etc
```

Comment:
Are you sure this is where it goes? If it is a tracker add-on, then the proper place would be /boot/home/config/non-packaged/add-ons/Tracker